plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -134,6 +134,9 @@ fu_thunderbolt_device_get_version (FuThunderboltDevice *self)
 	/* read directly from file to prevent udev caching */
 	g_autofree gchar *safe_path = g_build_path ("/", self->devpath, "nvm_version", NULL);
 
+	if (!g_file_test (safe_path, G_FILE_TEST_EXISTS))
+		return FALSE;
```

Comment:
I think if there are multiple points of failure then we should perhaps add a `GError` to `fu_thunderbolt_device_get_version` and print the failure in the caller rather than just the generic `failed to read version` string.

Reply:
Sounds like a good idea, I'll rework it.